website/client/src/components/header/notifications/g1g1.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
```

Comment:
since most of the strings here are static a few `v-once` will help with performances

Reply:
OK! Does that cause trouble when users change their locale preference, though?

Reply:
I don't know if this helps, but when you change your language at User Icon > Settings, the website reloads itself.

Reply:
yes, since the site is reloaded when the locale changes there should be no issues